src/server/graphql/types/CreateUserPicturePutUrlPayload.js

```diff
@@ -0,0 +1,13 @@
+import {GraphQLObjectType} from 'graphql';
+import GraphQLURLType from 'server/graphql/types/GraphQLURLType';
+
+const CreateUserPicturePutUrlPayload = new GraphQLObjectType({
```

Comment:
Hahaha oh man, i hear you! I had some good talks with the GraphQL folks about this about a year ago.

Eventually, they won me over with 2 good arguments:
1. it's more expressive:
```
createOrgPicturePutUrl(args) {
      url
    }

VS.

createOrgPicturePutUrl(args) 
```
In the former, you are asking to get a url back. In the latter, you're asking a scalar result.

2. Future proof
Let's say I want to return 2 things. _now_ my type has changed & all my clients need to update their APIs because it'll be immediately broken (remember, GraphQL schema versioning is an antipattern!)
If I return an object, I can add fields to the payload & everyone can keep on trucking by just ignoring the new field.

Reply:
Yeah, the future proof part is what I was thinking too. Cool!